packages/main/src/ColorPalette.hbs

```diff
@@ -1,13 +1,26 @@
-<div class="ui5-cp-item-container" 
-	role="region"
-	aria-label="{{colorContainerLabel}}"
-	@click={{_onclick}}
-	@keyup={{_onkeyup}}
-	@keydown={{_onkeydown}}>
-	{{#each this.displayedColors}}
-		<slot
-			name="{{this._individualSlot}}"
-		>
-		</slot>
-	{{/each}}
+<div class="ui5-cp-root">
+	<div class="ui5-cp-item-container" 
+		role="region"
+		aria-label="{{colorContainerLabel}}"
+		@click={{_onclick}}
+		@keyup={{_onkeyup}}
+		@keydown={{_onkeydown}}>
+		{{#each this.displayedColors}}
+			<slot
+				name="{{this._individualSlot}}"
+			>
+			</slot>
+		{{/each}}
+	</div>
+
+	{{#if moreColors}}
```

Comment:
Perhaps checking if the moreColors property is set to true and the feature is imported/enabled is more accurate. You can combine these 2 conditions in one getter and use it here.